mllib/src/main/scala/org/apache/spark/mllib/fpm/AssociationRules.scala

```diff
@@ -107,7 +124,8 @@ object AssociationRules {
       @Since("1.5.0") val antecedent: Array[Item],
       @Since("1.5.0") val consequent: Array[Item],
       freqUnion: Double,
-      freqAntecedent: Double) extends Serializable {
+      freqAntecedent: Double,
+      freqConsequent: Option[Long]) extends Serializable {
```

Comment:
Wouldn't we now always know the frequency of the consequent?

Reply:
now we do, but for existing application using old methods it may not be available.